scripts/surface_charge/ReadMe.md

```diff
@@ -0,0 +1,48 @@
+# Surface Charge Calculator
+
+## Summary
+
+This tool returns the surface charge for a given structure and list of supplied hkl and offsets.
+The script provides a GUI that can be used from Mercury or from the command line.
+
+The output is a HTML file with a table for all the all selected surfaces and their associated charge, projected surface areas and normalised surface charge.
+
+Charges are currently calculated using Gasteiger charges. Further development could be made to use user derived charges. Please let me know if that is something you'd like [amoldovan@ccdc.cam.ac.uk](amoldovan@ccdc.cam.ac.uk).
+
+Example Output:
+![Example Output](assets/example_output.png)
+
+> **Note** - When comparing charges for structures out of the CSD and from mol2 files the values might be different as the bonding might not be the same. When importing a mol2 the bonding and charges have to be calculated on the fly. Whereas the CSD structures the bonding is pre-assigned.
```

Comment:
Whereas for CSD structures?

Reply:
Yes thank you! I got sidetracked whilst writing this note and it could probably do with re-writing anyway